functions/ts/package.json

```diff
@@ -13,7 +13,7 @@
     "lint-license": "license-checker --onlyAllow 'Apache-2.0;MIT;BSD;BSD-2-Clause;BSD-3-Clause;ISC;CC-BY-3.0;CC0-1.0;Unlicense'",
     "format": "prettier \"src/**\" \"*.json\" --write",
     "pretest": "npm run build",
-    "test": "jasmine --config=jasmine.json",
+    "test": "nyc -x 'src/gen/**' -x '**/*_test.ts' jasmine --config=jasmine.json",
```

Comment:
Let's keep the test framework consistent across functions in the catalog and the sdk repo. Any reason you're changing this?

Reply:
I want to know the coverage of my unit test codes. Our current unit tests are really simple and I even have no idea how simple they are.

Reply:
By this tool I can see a report of the unit test coverage. Like this
```
---------------------|---------|----------|---------|---------|--------------------------
File                 | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s        
---------------------|---------|----------|---------|---------|--------------------------
All files            |   67.98 |    56.98 |    62.5 |    68.5 |                          
 helm_template.ts    |   98.68 |    93.33 |     100 |   98.67 | 121                      
 istioctl_analyze.ts |   55.26 |    33.33 |      60 |   55.26 | 46-89,103,111-112        
 kubeval.ts          |    54.1 |    54.17 |      20 |      55 | 50,61-93,115-116,140-152 
 kustomize_build.ts  |   32.14 |        0 |      50 |   33.33 | 26-39,51-62              
---------------------|---------|----------|---------|---------|--------------------------
```

Reply:
Couple of things:

1. I understand, but we want to have a consistent dev environment across our NPM packages. So we should either use jasmine everywhere or nyc everywhere.

2. Also keep in mind my other comment about evaluating new dependencies to avoid pulling in a transitive deps. For maintainability and security reasons, you want as few transitive deps as possible.

For comparison:

http://npm.broofa.com/?q=jasmine

vs

http://npm.broofa.com/?q=nyc

The latter pulls in A LOT of deps. That's the reason we decided to stick with jasmine.